drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/s3_zip_files_importer.rb

```diff
@@ -45,36 +45,34 @@ def self.mper
     def run!
       s3 = remote_credential&.s3 || AwsS3.new(bucket_name: bucket_name)
 
-      s3.list_objects(prefix: prefix).each do |s3_object|
-        unless s3_object.key.match?(/.zip$/i)
-          Rails.logger.debug "Skipping a non-zip file #{s3_object.key}"
-          next
-        end
+      # Choose which file to import (most recent)
+      s3_object = s3.list_objects(prefix: prefix).first
```

Comment:
Does this want to be sorted in some way? I feel like it should be `max_by` instead of first.

Reply:
it should give the most recently modified, at least thats what I have seen in experience. we use `first` in other places too.

https://github.com/greenriver/hmis-warehouse/blob/3913be625384ad9183dadd45591121a98125ed61/app/models/aws_s3.rb#L72-L99